parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -80,63 +73,141 @@ impl pallet_referenda::Config<FellowshipReferendaInstance> for Runtime {
 	type DemoteOrigin = EitherOf<
-		frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>,
-		EitherOf<
-			MapSuccess<
-				EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
-				Replace<ConstU16<9>>,
-			>,
-			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<2>>>,
+		frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<{ ranks::DAN_9 }>>,
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<{ ranks::DAN_9 }>>,
 		>,
 	>;
 	type Polls = FellowshipReferenda;
-	type MinRankOfClass = sp_runtime::traits::Identity;
+	type MinRankOfClass = tracks::MinRankOfClass;
 	type VoteWeight = pallet_ranked_collective::Geometric;
 }
+
+pub type FellowshipCoreInstance = pallet_core_fellowship::Instance1;
+
+impl pallet_core_fellowship::Config<FellowshipCoreInstance> for Runtime {
+	type WeightInfo = weights::pallet_core_fellowship::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Members = pallet_ranked_collective::Pallet<Runtime, FellowshipCollectiveInstance>;
+	type Balance = Balance;
+	// Parameters are set by any of:
+	// - Root;
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote among all Fellows.
+	type ParamsOrigin = EitherOfDiverse<
+		EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+		Fellows,
+	>;
+	// Induction (creating a candidate) is by any of:
+	// - Root;
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a single Fellow;
+	// - a vote among all Members.
+	type InductOrigin = EitherOfDiverse<
+		EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+		EitherOfDiverse<
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				FellowshipCollectiveInstance,
+				{ ranks::DAN_3 },
+			>,
+			Members,
+		>,
+	>;
+	// Approval (rank-retention) of a Member's current rank is by any of:
+	// - Root;
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote by the rank two above the current rank for all retention up to the Master rank.
+	type ApproveOrigin = EitherOf<
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<{ ranks::DAN_9 }>>,
+		>,
+		EnsureCanRetainAt,
+	>;
+	// Promotion is by any of:
+	// - Root can promote arbitrarily.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote by the rank two above the new rank for all promotions up to the Master rank.
+	type PromoteOrigin = EitherOf<
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<{ ranks::DAN_9 }>>,
+		>,
+		EnsureCanPromoteTo,
+	>;
+	type EvidenceSize = ConstU32<65536>;
+}
+
+pub type FellowshipSalaryInstance = pallet_salary::Instance1;
+
+use xcm::prelude::*;
+
+parameter_types! {
+	pub Statemint: MultiLocation = (Parent, Parachain(1000)).into();
+	pub StatemintUsdtId: AssetId = GeneralIndex(1984).into();
```

Comment:
I think you will need to say under which asset pallet (index 50) this ID falls?
```suggestion
	pub AssetHubLocation: MultiLocation = (Parent, Parachain(1000)).into();
	pub AssetHubUsdtId: AssetId = GeneralIndex(1984).into();
```

Reply:
yes, you are right, pallet index is expected, checked with xcm-emulator